### PR TITLE
feat(addr):No way to limit client and REST APIs to localhost

### PR DESCRIPTION
**Pull request by EMQmyd**

### CODE REVIEW

File: etc/kuiper.yaml [hunk 1]
```diff
@@ -9,9 +9,13 @@ basic:
   rotateTime: 24
   # Maximum file storage hours
   maxAge: 72
+  # CLI ip
+  ip: "0.0.0.0"
   # CLI port
   port: 20498
   # REST service port
```
Comment (jinfahua) [inline etc/kuiper.yaml:16]: Rest service bind address

File: etc/kuiper.yaml [hunk 2]
```diff
@@ -9,9 +9,13 @@ basic:
   rotateTime: 24
   # Maximum file storage hours
   maxAge: 72
+  # CLI ip
```
Comment (jinfahua) [inline etc/kuiper.yaml:12]: CLI service bind address

File: etc/kuiper.yaml [hunk 3]
```diff
@@ -9,9 +9,13 @@ basic:
...
+  ip: "0.0.0.0"
```
Comment (jinfahua) [inline etc/kuiper.yaml:13]: I don't think double quotations are necessary, could you please verify it?

**Review by jinfahua — CHANGES_REQUESTED** [review]

**Comment by ngjaying** [comment]
Is it better to follow docker compose's rule for [ports](https://docs.docker.com/compose/compose-file/#ports). 
```
port: 9081                     -> if no ip specified, defaults to 0.0.0.0; It is also backward compatible
port: 127.0.0.1:9081
```

**Pull request closed without merging**